help, lost!

ztmurphy21 opened the pull request:
i made a for loop for the readfile and a constructor in the one class. I
am lost on how to do this.

Commit:
help, lost!

i made a for loop for the readfile and a constructor in the one class. I
am lost on how to do this.

Commit:
a little bit of progress. i think..

Commit:
making some progress, but still error message

re to comments.

Commit:
debug

Commit:
now outputing the file

just need to tokenize.

Commit:
starting to tokenize

did a little bit with tokenizing, but stuck on it have an error message.

ninjapanzer commented on src/dnddiceroller/DnDDiceRoller.java:
```diff
@@ -0,0 +1,80 @@
+      Scanner scanner = null;
+      ArrayList<String> list = new ArrayList<String>();
+        File temp = new File ("dice.txt");
+        Scanner file = new Scanner (temp);
```
You assigned this scanner to `file` instead of `scanner` so it can't read any lines

ninjapanzer commented on src/dnddiceroller/DnDDiceRoller.java:
```diff
@@ -0,0 +1,80 @@
+        Scanner file = new Scanner (temp);
+        String s = scanner.nextLine();
+   while(file.hasNext()){
+       String[] tempList=s.split(":");
```
Before you deal with tokenizing the file you will need to exclude the lines of the file that do not have useful information starting with `####`

ninjapanzer replied:
You also want to tokenize the `" "`'s before you split on the `":"`

ninjapanzer commented on src/dnddiceroller/DnDDiceRoller.java:
```diff
@@ -0,0 +1,80 @@
+       String title = tempList[0];
+       String type = tempList[1].substring(1); 
+       list.add(title);
+       s = scanner.nextLine();
```
You are probably going to want to capture you nextLine at the beginning of your loop or you will lose the last line

Commit:
another try

still has some issues,

Commit:
removing the #### from the file

removed the #### from the file. what else should I do. This may come
across as other changes but this was the only change bc it was copied
over to my surface.
